clients/drcachesim/tracer/raw2trace.h

```diff
@@ -672,6 +679,103 @@ struct trace_header_t {
             return msg;    \
     } while (0)
 
+template <typename T> class bitset_hash_table_t {
+private:
+    const static size_t BLOCK_SIZE_BIT = 13;
+    const static size_t BLOCK_SIZE = (1 << BLOCK_SIZE_BIT);
+    const size_t BASIC_BUCKET_COUNT = (1 << 15);
+    std::unordered_map<T, std::bitset<BLOCK_SIZE>> page_table;
```

Comment:
style: class member variable names end in _

Reply:
done #6176